support-config/src/main/scala/com/gu/support/config/ZuoraConfigProvider.scala

```diff
@@ -67,4 +74,9 @@ class ZuoraConfigProvider(config: Config, defaultStage: Stage)
     monthlyChargeId = config.getString("monthly.productRatePlanChargeId"),
     annualChargeId = config.getString("annual.productRatePlanChargeId"),
   )
+
+  private def invoiceTemplatesFromConfig(config: Config) = ZuoraInvoiceTemplatesConfig(
+    defaultTemplateId = config.getString("default"),
```

Comment:
Just a question on readability and Guardian standards: should `defaultTemplateId` map to `defaultId` with the addition of the `Id` suffix. Same for `au`